plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/helpContexts.xml

```diff
@@ -2,15 +2,15 @@
 <contexts>
   <context id="DeployAppEngineStandardProjectContext" title="App Engine Deployment - Standard Environment">
     <!-- Eclipse requires escaped pseudo-HTML here -->
-    <description>Properties used when deploying the project to the App Engine standard environment.
+    <description>Properties used when deploying the project to the App Engine Standard environment.
```

Comment:
I believe "standard" is customarily lower case in non-title text. See https://cloud.google.com/appengine/docs/standard/ for example